app/scenes/Document/components/EditableTitle.tsx

```diff
@@ -128,14 +170,31 @@ const EditableTitle = React.forwardRef(
         onBlur={onBlur}
         placeholder={placeholder}
         value={value}
-        $emojiWidth={emojiWidth}
         $isStarred={document.isStarred}
+        $containsEmoji={!!document.emoji}
+        $isFocused={isFocused}
         autoFocus={!document.title}
         maxLength={DocumentValidation.maxTitleLength}
         readOnly={readOnly}
         dir="auto"
         ref={ref}
       >
+        {(!isMobile || isFocused || document.emoji) && (
+          <EmojiPicker
```

Comment:
We're showing this edit UI to users that don't have edit permissions, the button to choose should not show on hover for viewers